src/utils.ts

```
@@ -34,14 +34,64 @@ const enforceRole = (header: string | undefined): string => {
 };
+
+const sanitize = (value: any, options: SanitizeOptions) => {
+  if (value == null || value === "") {
+    return options.defaultValue;
+  }
+
+  switch (options.type) {
+    case "number": {
+      const num = Number(value);
+      if (
+        isNaN(num) ||
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)